src/appleseed.studio/python/pythoninterpreter.cpp

```diff
@@ -106,6 +106,26 @@ namespace
 
         return lib_path.string();
     }
+
+    string compute_bundled_plugins_path()
```

Comment:
Can't we use `application::shared::Application::get_root_path()` instead of reimplementing the logic here?